assistant/gpt_assistant.py

```diff
@@ -416,7 +417,10 @@ async def send_to_assistant(
         # Inject context into our copy by appending it to system message. Unclear whether multiple
         # system messages are confusing to the assistant or not but cursory testing shows this
         # seems to work.
-        extra_context_message = Message(role=Role.SYSTEM, content=self._create_context_system_message(local_time=local_time, location=location_address, learned_context=learned_context))
+        additional_context = self._create_context_system_message(local_time=local_time, location=location_address, learned_context=learned_context)
+        if noa_system_prompt is not None:
+            additional_context = f"{noa_system_prompt}\n{additional_context}"
+        extra_context_message = Message(role=Role.SYSTEM, content=additional_context)
```

Comment:
How long are these prompts expected to be? Should they be appended as an extra system message or in the primary system message? Note that Claude does not allow multiple system messages.

Reply:
Its for Noa app specific like taking a subtle character.
for claude we can append in system prompt